Compilation fails under Cygwin

```
#=== ERROR while compiling spawn.v0.15.0 ======================================#
# context     2.0.3 | cygwin/x86_64 | ocaml-base-compiler.4.13.1 | https://opam.ocaml.org#80d7a1e3
# path        ~/.opam/4.13.1/.opam-switch/build/spawn.v0.15.0
# command     ~/.opam/4.13.1/bin/dune build -p spawn -j 7 @install
# exit-code   1
# env-file    ~/.opam/log/spawn-11344-e22673.env
# output-file ~/.opam/log/spawn-11344-e22673.out
### output ###
#          gcc src/spawn_stubs.o (exit 1)
# (cd _build/default/src && /bin/gcc -O2 -fno-strict-aliasing -fwrapv -pthread -D_FILE_OFFSET_BITS=64 -U_WIN32 -O2 -fno-strict-aliasing -fwrapv -pthread -g -I /home/mobaxterm/.opam/4.13.1/lib/ocaml -I /home/mobaxterm/.opam/4.13.1/lib/ocaml/threads -o spawn_stubs.o -c spawn_stubs.c)
# spawn_stubs.c:32:10: fatal error: sys/syscall.h: No such file or directory
#  #include <sys/syscall.h>
#           ^~~~~~~~~~~~~~~
# compilation terminated.
```

I have not tried to dig into it in any way (and in particular, not sure where the `-U_WIN32` is coming from)... No clue whether this is even intended/supposed to work in the first place.

--- a/src/spawn_stubs.c
+++ b/src/spawn_stubs.c
@@ -29,7 +29,9 @@ CAMLprim value spawn_is_osx()
 
 #include <assert.h>
 #include <string.h>
+#if !defined(__CYGWIN__)
 #include <sys/syscall.h>
+#endif
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <unistd.h>

